platform/ios/src/MGLMapView.mm

```diff
@@ -4089,8 +4089,12 @@ - (MGLAnnotationTag)annotationTagAtPoint:(CGPoint)point persistingResults:(BOOL)
             }
             else
             {
-                if ([annotation isKindOfClass:[MGLMultiPoint class]])
+                if ([annotation conformsToProtocol:[MGLShape self]])
```

Comment:
`MGLShape` isn't a protocol (unless there's also a protocol with the same name?) - so this check should be a `isKindOfClass` test.

However, this relates to my question above - perhaps it is worth having an "interactable annotation" protocol after all?